### DISCUSSION TITLE
Horizontal navigation menu

### DISCUSSION THREAD

**Discussion by essadek** [fabiocaccamo/django-admin-interface#330] [Q&A]
I'm using the theme for a new project.
I have always chosen to implement horizontal nav menu for my previous projects using other themes, so my question is: does **django-admin-interface** offer it as builtin feature or I have to implement it myself ?

**Comment by fabiocaccamo** [comment]
@essadek actually there is not horizontal menu support (maybe there will be in the future), in the meantime you can add it by using some third-party package. 

There is already this issue where you can find more info: https://github.com/fabiocaccamo/django-admin-interface/issues/125
